scipy/stats/_continuous_distns.py

```diff
@@ -5392,8 +5392,165 @@ def _updated_ctor_param(self):
         return dct
 
 
+def _get_shape_parameters(distribution):
+    """
+    Returns all shape parameters of a scipy distribution
+    including loc and scale
+    @param distribution scipy.stats.rv_continuous
+    """
+    shape_parameters = []
+    if distribution.shapes is not None:
+        shape_parameters += distribution.shapes.split(', ')
+    shape_parameters += ['loc', 'scale']
```

Comment:
If I remeber correctly the reason is that loc and scale must be the last parameters because at least at some points in the scipy.stats code the arguments to a distirbution are passed as args (not as kwargs), so the order matters, and the order is "shape_parameters, loc, scale".